src/poetry/console/commands/add.py

```diff
@@ -263,6 +269,9 @@ def handle(self) -> int:
             if self.option("platform"):
                 constraint["platform"] = self.option("platform")
 
+            if self.option("markers"):
```

Comment:
Call to `self.option` could be walrusified since it's not cached and can be quite expensive in long commands. However, since we don't do that globally anyway, feel free to ignore that.

Reply:
Than I choose to ignore :smile: